src/python/WMCore/Storage/StageOutMgr.py

```diff
@@ -364,40 +384,10 @@ def cleanSuccessfulStageOuts(self):
             msg += "Using command implementation: %s\n" % command
             logging.info(msg)
             delManager = DeleteMgr(**self.overrideConf)
+            delManager.bypassImpl = self.bypassImpl
             try:
                 delManager.deletePFN(pfn, lfn, command)
             except StageOutFailure as ex:
                 msg = "Failed to cleanup staged out file after error:"
                 msg += " %s\n%s" % (lfn, str(ex))
                 logging.error(msg)
-
-    def searchTFC(self, lfn):
```

Comment:
Has this one been superseeded by `StageOutMgr.searchRFC` as well?

Reply:
Yes